src/providers/index.ts

```diff
@@ -17,18 +17,20 @@ import Web3 from 'web3'
 import {
   GenericMappedTokenData,
   PolygonMappedTokenData,
-  BnbMappedTokenData,
-  BnbMappedToken,
+  MappedTokenData,
+  MappedToken,
 } from '../constants/types'
+import { AvalancheMappingProvider } from './AvalancheMappingProvider'
 
 const web3 = new Web3()
 
 // chains we support fetching mappings for (can be different than the l2ChainIds arg for buildList)
-const SUPPORTED_L2_CHAINS = [
+const SUPPORTED_CHAINS = [
```

Comment:
Supported chains of what? It seems like before it was L2 related, but what do we use this constant for?

Reply:
talked to matteen about renaming this - the L2 name is outdated actually since we added BNB. basically it indicates the chains we have "....MappingProvider.ts" files for in this repo, like AvalancheMappingProvider.ts. Maybe this new name "CHAINS_WITH_MAPPING_PROVIDERS" is more clear?